test/index.test.js

```diff
@@ -244,4 +213,18 @@ describe('Live Index Tests', () => {
     });
     assert.strictEqual(res.statusCode, 200);
   }).timeout(20000);
+  condit('Publish spark-website', condit.hasenvs(['AWS_S3_REGION', 'AWS_S3_ACCESS_KEY_ID', 'AWS_S3_SECRET_ACCESS_KEY']), async () => {
```

Comment:
why condit? I would try to remove the token for the polly recording.

Reply:
This is a *live* test that I sometimes run manually to test functionality, with my credentials.

Reply:
I'm always putting `(online)` or `#online` into the test name so that Tobi does not get too alerted.

Reply:
but why not record the test with polly?

Reply:
> but why not record the test with polly?

I execute these live (or online) tests against the real system, with varying parameter combinations. When I see a new behaviour that I need to handle, I create a test in the section above, and record it if polly (as happened with handling a missing source in a publish operation).

Reply:
@dominique-pfister so those are _only personal_ tests, and you don't expect the CI system to run them?

Reply:
yes

Reply:
yes -> those are only personal test? i.e. the CI will publish _random_ things ? 

I think it would be better to move them to a location there they don't get executed by default.
or tag and exclude them with `-ig dominique-tests`

Reply:
> because in the CI environment, those envs are defined

no, they're not: I just checked by re-running the CI tests and the `junit/test-results.xml` says:

```
<testcase classname="Live Index Tests" name="Store theblog (env vars AWS_S3_REGION, AWS_S3_ACCESS_KEY_ID, AWS_S3_SECRET_ACCESS_KEY must be set)" time="0"><skipped/></testcase>
<testcase classname="Live Index Tests" name="Store spark-website (env vars AWS_S3_REGION, AWS_S3_ACCESS_KEY_ID, AWS_S3_SECRET_ACCESS_KEY must be set)" time="0"><skipped/></testcase>
<testcase classname="Live Index Tests" name="Publish spark-website (env vars AWS_S3_REGION, AWS_S3_ACCESS_KEY_ID, AWS_S3_SECRET_ACCESS_KEY must be set)" time="0"><skipped/></testcase>
<testcase classname="Live Storage Tests" name="Read from code bus (env vars AWS_S3_REGION, AWS_S3_ACCESS_KEY_ID, AWS_S3_SECRET_ACCESS_KEY must be set)" time="0"><skipped/></testcase>
<testcase classname="Live Storage Tests" name="Read metadata from content bus (env vars AWS_S3_REGION, AWS_S3_ACCESS_KEY_ID, AWS_S3_SECRET_ACCESS_KEY must be set)" time="0"><skipped/></testcase>
<testcase classname="Live Storage Tests" name="Copy object in content bus (env vars AWS_S3_REGION, AWS_S3_ACCESS_KEY_ID, AWS_S3_SECRET_ACCESS_KEY must be set)" time="0"><skipped/></testcase>
```

Reply:
> no, they're not: I just checked by re-running the CI tests

ok ok.... :-) but this is not guaranteed. or what if another developer has those defined and then running the tests would modify production data ?

In any case, it would be better to isolate them better.

Reply:
I agree: is always better to do something better 😄

Reply:
Jokes aside: I agree that this might be dangerous, so I'll add a skip to those production tests' describe